OpenPanelSDK/src/main/java/com/dev/openpanelsdk/OpenPanel.kt

```diff
@@ -199,6 +200,9 @@ class OpenPanel(private val context: Context, private val options: Options) {
     private fun getDefaultEventProperties(): Map<String, Any> {
         val ret = mutableMapOf<String, Any>()
 
+        ret["__brand"] = Build.BRAND ?: "UNKNOWN"
```

Comment:
Skip the "UNKNOWN" since this will override anything else (its truthy). Would want `undefined` or the equal counterpart in kotlin

Reply:
What if I just add `--` instead of unknown!